packages/client/hooks/useUsageSnackNag.ts

```diff
@@ -0,0 +1,77 @@
+import graphql from 'babel-plugin-relay/macro'
+import ms from 'ms'
+import {useEffect} from 'react'
+import {useLazyLoadQuery} from 'react-relay'
+import {RouterProps, useHistory} from 'react-router'
+import Atmosphere from '../Atmosphere'
+import SendClientSegmentEventMutation from '../mutations/SendClientSegmentEventMutation'
+import {TierEnum} from './../__generated__/NewTeamOrgPicker_organizations.graphql'
+import {useUsageSnackNagQuery} from './../__generated__/useUsageSnackNagQuery.graphql'
+import useAtmosphere from './useAtmosphere'
+
+const NAG_EVERY = ms('3m')
+
+const getIsNaggingPath = (history: RouterProps['history']) => {
+  const {location} = history
+  const {pathname} = location
+  return !(pathname.includes('/usage') || pathname.includes('/meet/'))
+}
+
+const shouldNag = (tier: TierEnum, suggestedTier: TierEnum | null) => {
+  if (!suggestedTier) return false
+  const suggestPro = suggestedTier === 'pro' && tier === 'personal'
+  const suggestEnterprise = suggestedTier === 'enterprise' && tier !== 'enterprise'
+  return suggestPro || suggestEnterprise
+}
+
+const emitNag = (atmosphere: Atmosphere, history: RouterProps['history']) => {
+  atmosphere.eventEmitter.emit('addSnackbar', {
+    key: 'usage',
+    message: 'View your usage stats',
+    autoDismiss: 0,
+    onDismiss: () => {
+      setTimeout(() => {
+        const isNaggingPath = getIsNaggingPath(history)
+        if (!isNaggingPath) return
+        emitNag(atmosphere, history)
+      }, NAG_EVERY)
+    },
+    action: {
+      label: 'View Usage',
```

Comment:
I wanted the CTA to take the user to the `/usage` page rather than being a `Dismiss` button like the other snack nag because it saves the user a couple of clicks, and the `/usage` page is somewhat hidden.

I considered changing the copy to `Checkout Usage` as `View Usage` doesn't feel super friendly / a bit boring, but users might associate `Checkout` with a shopping cart, so I kept it as `View`.